### DISCUSSION TITLE
Update in pyjwt breaks generate_jwt in utils.py

### DISCUSSION THREAD

**Discussion by Itayo252** [fastapi-users/fastapi-users#1088] [Q&A] [answered]
About two weeks ago, a new release of ```pyjwt``` dropped and it changed the return value (and more specifically the type of the return value) from bytes to str. According to the old implementation, the code uses ```.decode("utf-8")``` on the return value to convert the bytes to a string. In the new version, the library returns the string and o decoding is needed' however this causes a problem since calling ```.decode()``` on a string raises an error.

Can somebody please check it out? I'm afraid I'm missing something and if not the fix is simple (deleting the redundant decoding).

**Comment by frankie567** [comment, marked as answer]
Which version of FastAPI Users are you using? Currently, the JWT implementation looks like this and should work properly, even with PyJWT 2.5.0:

https://github.com/fastapi-users/fastapi-users/blob/a665cd5ed731679b342ec9c2869175447358ddba/fastapi_users/jwt.py#L30-L41

**Reply by Itayo252** [reply]
Sorry for the trouble, it's my bad. PyCharm didn't find the package so I used an ancient and non-compatible version, all good now.
